src/runtime-rs/crates/runtimes/virt_container/src/container_manager/container_inner.rs

```diff
@@ -183,6 +184,17 @@ impl ContainerInner {
                 }
             })?;
 
+        self.agent
+            .drop_cache(DropCacheRequest {
+                drop_option: 2,
+                ..Default::default()
```

Comment:
Thx @xyc-grace 
Some comments about the reason of  drop_option(2) is needed here.

Reply:
I got the same question here.